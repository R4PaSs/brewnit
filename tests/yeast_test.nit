import dummy_db

with ctx = new DBContext do
	var yeasts = ctx.yeast_worker.fetch_multiple(ctx, "* FROM yeasts")
	for i in yeasts do print i
end
