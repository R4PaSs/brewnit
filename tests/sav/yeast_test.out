Yeast Safale, S-05
Yeast White Labs, WLP400; Aliases: Belgian Wit Ale Yeast
Yeast White Labs, WLP013; Aliases: London Ale Yeast
